Content.Server/Sticky/Systems/StickySystem.cs

```diff
@@ -20,17 +21,48 @@ public sealed class StickySystem : EntitySystem
     [Dependency] private readonly SharedHandsSystem _handsSystem = default!;
     [Dependency] private readonly SharedInteractionSystem _interactionSystem = default!;
     [Dependency] private readonly SharedAppearanceSystem _appearance = default!;
+    [Dependency] private readonly EntityLookupSystem _lookup = default!;
 
     private const string StickerSlotId = "stickers_container";
 
     public override void Initialize()
     {
         base.Initialize();
+        SubscribeLocalEvent<TransferStickySurfaceOnSpawnBehaviorComponent, OnSpawnFromSpawnEntitiesBehaviourEvent>(TransferStickySurfaceFromDestroyedEntity);
+
+        SubscribeLocalEvent<StickyComponent, MapInitEvent>(OnMapInit);
         SubscribeLocalEvent<StickyComponent, StickyDoAfterEvent>(OnStickFinished);
         SubscribeLocalEvent<StickyComponent, AfterInteractEvent>(OnAfterInteract);
         SubscribeLocalEvent<StickyComponent, GetVerbsEvent<Verb>>(AddUnstickVerb);
     }
 
+    private void OnMapInit(EntityUid uid, StickyComponent component, MapInitEvent args)
+    {
+        if (!component.StickOnStart || component.StuckTo != null)
```

Comment:
if StuckTo is not null on mapinit, it should automatically stick, not return and do nothing

Reply:
doesn't `StuckTo` is not null mean that it already stuck? I don't think it needs to restick in that case